### PR TITLE
Accessibility: Case details close button popup in web apps

### PR DESCRIPTION
**Pull request by akashkj**
## Product Description
Add accessibility label for close button in Case details pop-up.

## Technical Summary
Added `aria-label` to close button HTML in case details 
https://dimagi-dev.atlassian.net/browse/USH-1936

## Feature Flag
NA

### Safety story
Tested in local environment using Linux's native screen reader

https://user-images.githubusercontent.com/14318173/214170825-fb5c3b10-9a6e-4af5-a600-de10c70519b4.mp4

### Automated test coverage

NA

### QA Plan
Pending

### Migrations
NA

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/cloudcare/templates/formplayer/case_detail.html [hunk 1]
```diff
@@ -51,7 +51,7 @@
   <div class="modal-dialog">
     <div class="modal-content">
       <div class="modal-header">
-        <button type="button" class="close" data-dismiss="modal">&times;</button>
+        <button type="button" aria-label="Close" class="close" data-dismiss="modal">&times;</button>
```
Comment (orangejenny) [inline corehq/apps/cloudcare/templates/formplayer/case_detail.html:54]: translate

Comment (akashkj) [reply corehq/apps/cloudcare/templates/formplayer/case_detail.html]: https://github.com/dimagi/commcare-hq/pull/32521/commits/c2de4eac0917504e075eba16f9f3e6dc3394f757

**Comment by akashkj** [comment]
@orangejenny Can this be merged or require additional reviewers?

**Review by orangejenny — APPROVED** [review]

**Comment by orangejenny** [comment]
@akashkj thanks for pinging, this is good to go

**Pull request merged**